time.go

```diff
@@ -12,3 +12,11 @@ const (
 var (
 	TimeBaseQ = newRationalFromC(C.AV_TIME_BASE_Q)
 )
+
+func Q2D(a Rational)  float64{
```

Comment:
What is the difference between [this function](https://github.com/asticode/go-astiav/blob/master/rational.go#L40) and the function you've added? 🤔  (I've renamed the function in `go-astiav` from `ToDouble` to `Float64` since that may be clearer)

Reply:
Your approach looks better！:satisfied: